heatmiserv3/heatmiser.py

```diff
@@ -366,8 +366,13 @@ def get_program_mode(self):
         modes = {0: "5/2 mode", 1: "7 day mode"}
         return modes[mode]
 
-    def get_frost_protection(self):
-        pass
+    def get_frost_protection(self) -> bool:

```

Comment:
Sorry it's taken me so long.

Unfortunately this value won't be a bool; as one model of thermostat also returns cooling mode, 

Run mode
00: heating mode
01: frost protection mode
02: cooling mode